plugins/modules/vmware_host_firewall_manager.py

```diff
@@ -381,16 +387,27 @@ def main():
     )
 
     for rule_option in module.params.get("rules", []):
-        if 'allowed_hosts' in rule_option:
-            if isinstance(rule_option['allowed_hosts'], list):
-                if len(rule_option['allowed_hosts']) == 1:
-                    allowed_hosts = rule_option['allowed_hosts'][0]
-                    rule_option['allowed_hosts'] = allowed_hosts
-                    module.deprecate(
-                        msg='allowed_hosts should be a dict, not a list',
-                        version='3.0.0',
-                        collection_name='community.vmware'
-                    )
+       if 'allowed_hosts' in rule_option:
+           if isinstance(rule_option['allowed_hosts'], list):
+               if len(rule_option['allowed_hosts']) == 1:
+                   allowed_hosts = rule_option['allowed_hosts'][0]
+                   rule_option['allowed_hosts'] = allowed_hosts
+                   module.deprecate(
+                       msg='allowed_hosts should be a dict, not a list',
+                       version='3.0.0',
+                       collection_name='community.vmware'
+                   )
+       if not rule_option.get("enabled"):
+           continue
+       try:
+           isinstance(rule_option["allow_hosts"]["all_ip"], bool)
```

Comment:
This still says `allow_hosts` and not `allowed_hosts` (https://github.com/ansible-collections/community.vmware/pull/564#discussion_r541192619).

Reply:
oh... thank you.